### ISSUE TITLE
For legacy browsers (without modules)

### ISSUE THREAD

**Issue by clarkk** [inorganik/countUp.js#309]
When I try to use the legacy version without modules I get an error

```
<script src="/js/github/inorganik/countUp.js/countUp.withPolyfill.min.js" defer></script>
```

Error
```
caught SyntaxError: Unexpected token 'export' (at countUp.withPolyfill.min.js:1:6208)
```

Chrome Version 112.0.5615.86 (Officiel version) (64-bit)

**Comment by inorganik** [comment]
I checked and that file is an es6 module, doh. How are you trying to import countup?

**Comment by clarkk** [comment]
Just as in the 90'es like

```
<script src="/js/github/inorganik/countUp.js/countUp.withPolyfill.min.js" defer></script>
```

Without any kind of module import

**Comment by inorganik** [comment]
Since it's a module, it doesn't expose a global so won't work like using
scripts in the 90's. I should probably fix that and package it up as UMD if
that works.

**Comment by clarkk** [comment]
Lookinng forward to a fix. Browsers without ES6 support is still around

**Comment by inorganik** [comment]
@clarkk Please try using `dist/countUp.umd.js` following the usage instructions in the readme, and include the polyfill just by copying it from the root of this repo - `requestAnimationFrame.polyfill.js`. If that works for you, I think I will just discontinue `countUp.withPolyfill.min.js`. Let me know.

**Comment by clarkk** [comment]
I just use umd without module and it works fine

```
<script src=".../countUp.umd.js"></script>

$('.anim-countup').each(function(){
	const anim = new countUp.CountUp(this, $(this).data('count'), {
		duration: 3,
		separator: '.',
		decimal: ',',
		easingFn(t, b, c, d){
			const ts = (t /= d) * t, tc = ts * t;
			return b + c * (tc + -3 * ts + 3 * t);
		}
	});
	if(!anim.error) anim.start();
	else console.error(anim.error);
});
```

**Comment by inorganik** [comment]
Great! Here's what the dist folder will look like after my change in #311 

```
dist/
├─ countUp.d.ts
├─ countUp.js
├─ countUp.min.js
├─ countUp.umd.js
├─ requestAnimationFrame.polyfill.js
```

so you'll just need to include the umd file and polyfill separately.

**Comment by inorganik** [comment]
This was merged and released in v2.6.1.
